todo_app/flask_config.py

```diff
@@ -1,8 +1,12 @@
 import os
 
-
 class Config:
     """Base configuration variables."""
     SECRET_KEY = os.environ.get('SECRET_KEY')
     if not SECRET_KEY:
         raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+    """TRELLO CREDENTIALS
+    TRELLO_API_KEY = os.environ.get('TRELLO_API_KEY')
+    TRELLO_API_TOKEN = os.environ.get('TRELLO_API_TOKEN')"""
```

Comment:
This is commented out, so why include it here? You could uncomment here and then access it via app.config rather than calling os.getenv directly in trello.py

Reply:
Tidied up commented out credentials